core/src/style/style.h

```diff
@@ -36,6 +35,14 @@ enum class Blending : int8_t {
     overlay,
 };
 
+struct StyledMesh {
```

Comment:
I agree another name would be nice, what about `DrawBuffer`? And maybe declare this in `vboMesh.h`.